src/components/GetDataDialog.tsx

```diff
@@ -19,10 +30,19 @@ function GetDataDialog({ open, onClose }: { open: boolean; onClose: () => void }
           Please follow the steps below
           <ol>
             <li>Select at least one dataset</li>
-            <li>Download the participant-level and dataset-level results files</li>
+            <li>Download the cohort participant machine results files using the button below</li>
             <li>Change directory to the location of the downloaded files</li>
-            <li>Copy and run the following command:</li>
+            <li>Copy and run the following command</li>
           </ol>
+        </DialogContentText>
+        <div className="mb-4 flex justify-center">
+          <DownloadResultButton
+            identifier="cohort participant machine"
```

Comment:
The issue with this is, the `btn` part and the `-` end up on the button in the UI because we're using the identifier for that. The use of `-` used to work because the words `participant-level` and `dataset-level` were already hyphenated.

Reply:
Ah, I see. I also misunderstood what this identifier does, I thought it was a cypress selector. Let's leave it! And if we ever get confused about the spaces, we can decouple identifier from button text then.